### PR TITLE
Feat: add About

### PR DESCRIPTION
**Pull request by danrocha**
Initial structure for the about page:

- Blurb on the companies list page, with a "read more" link
- About page with text

### CODE REVIEW

**Review by danrocha — COMMENTED** [review]
The idea of the about page is to tell a story, make it relatable and personal... IMHO I would leave the history in the first person, where you, Gui, tell the story of how it began.... then as we move through, we slowly change to "we" and finally introduce the people who are contributing. 

We still need to complement the text, I need some small help with that... maybe some bullet points I can elaborate on??

File: src/views/About.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="tos-container">
```
Comment (gtalarico) [inline src/views/About.vue:2]: tos ?
is just borrowed from terms of service page?

**Review by gtalarico — APPROVED** [review]
Looks, good will merge and do additional edits locally

**Comment by gtalarico** [comment]
Did some work on text, let me know what do you think @danrocha 
Also removed all custom css and added them to base typography styling since.


image
image

**Comment by danrocha** [comment]
I think it great. Just fixed some small typos and IMO it's ready to go

File: src/views/About.vue (hunk 1 above)
Comment (danrocha) [reply src/views/About.vue]: yes. it seemed tos styles were also used in the privacy page, so I went on and used it on the about page. therefore I thought we could have layout components (my Nuxt background wants layouts!!! haha)

**Comment by gtalarico** [comment]
will merge as is and we can make tweaks early next week before announcement if needed

**Pull request merged**